### PR TITLE
CT-1402 - Create storage token management endpoint tests

### PR DESCRIPTION
**Pull request by vojtabiberle**
Jira: [CT-14052](https://keboola.atlassian.net/browse/CT-1402)
Connection PR: https://github.com/keboola/connection/pull/5011

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated
- [x] You declared if there is a BC break or not (will affect next release of a client)

## Release

- [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

---

### CODE REVIEW

File: openapi.yml [hunk 1]
```diff
@@ -4289,6 +4287,10 @@ paths:
                     in.c:
                       type: string
                       description: 'main: read'
+                componentAccess:
```
Comment (tomasfejfar) [inline openapi.yml:4290]: Tohle nemá test, plus chybí test na to, že to spadne, když tam dám canManageTokens

**Review by tomasfejfar — CHANGES_REQUESTED** [review]

File: openapi.yml (hunk 1 above)
Comment (vojtabiberle) [reply openapi.yml:4290]: už jsou na oboje :-)

File: apiary.apib [hunk 2]
```diff
@@ -2125,12 +2125,12 @@ can be called with an application token with scope `manage:storage-tokens`.
 
     + description: Test Token (required) - Token description
     + canManageBuckets: true (boolean) - Token has full permissions on tabular storage
-    + canManageTokens: true (boolean) - Token has permission to create tokens in project
     + canReadAllFileUploads: true (boolean) - Token has full permissions to files staging
     + canPurgeTrash: true (boolean) - Allows permanently remove deleted configurations.
     + expiresIn: 60 (number) - Token lifetime
     + bucketPermissions (object)
         - in.c-main: read
+    + componentAccess: component1, component2 - Array of strings with component names
```
[the remark this answers is not in the document]
Comment (vojtabiberle) [reply apiary.apib]: fixed

**Comment by tomasfejfar** [comment]
Moment, sorry. Ten link je špatně

**Pull request merged**